### PR TITLE
Create a CONTRIBUTING.md

### LINKED ISSUE #110: Create a CONTRIBUTING.md
Creating a CONTRIBUTING.md for developers

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Resolves #110

### CODE REVIEW

File: CONTRIBUTING.md.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Contributing to Checkstyle Test Configurations Generator
```
Comment (romani) [inline CONTRIBUTING.md.txt:1]: Should be CONTRIBUTING.md
And under .github folder.

**Review by romani — CHANGES_REQUESTED** [review]
Item

File: CONTRIBUTING.md.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   ./setup.sh
+   ```
+
+## Making Changes
+
+1. Create a new branch for your changes:
+   ```
+   git checkout -b your-feature-branch
+   ```
+2. Make your changes in the `extractor` directory.
```
Comment (romani) [inline CONTRIBUTING.md.txt:24]: Please make a paragraph to mention that this repository can store any configuration bundle, any custom config and list of projects can be placed in special folder and stored in this repository. Names of files should be same as in all other folders.

**Review by romani — CHANGES_REQUESTED** [review]
Item

File: CONTRIBUTING.md.txt (hunk 2 above)
Comment (relentless-pursuit) [reply CONTRIBUTING.md.txt]: let me know if the changes are fine.

File: CONTRIBUTING.md.txt (hunk 1 above)
Comment (relentless-pursuit) [reply CONTRIBUTING.md.txt]: done

Comment (romani) [reply CONTRIBUTING.md.txt]: Not done

Comment (relentless-pursuit) [reply CONTRIBUTING.md.txt]: Sorry, i missed it. i guess done now.

**Review by romani — APPROVED** [review]

**Pull request merged**